src/Utils/StringUtils.cpp

```diff
@@ -188,7 +188,9 @@ void StringUtils::replaceInTokenVector(std::vector<std::string>& tokens,
       const bool surrounded_by_quotes =
           (i > 0 && (tokens[i - 1] == "\"")) &&
           ((i < tokensSize - 1) && (tokens[i + 1] == "\""));
-      tokens[i] = surrounded_by_quotes ? removeCR(news) : news;
+      std::string tmp(news);
```

Comment:
the original line would be working if we converted the second branch to string (as removeCR() returns a string
```
tokens[i] = surrounded_by_quotes ? removeCR(news)  : std::string(news);
```

I suspect the static analyzer was just tripped up by the fact that the two expressions are of different type.

Reply:
Just two different ways of keeping the compiler and the analyzer happy.

Reply:
Yeah, but  I think `?:` is the most readable way to express this. Up to you.

Reply:
Done!